css-canvas-medical/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <header class="row justify-between align-center border-bottom row1-marg font-a">
+      <img src="images/logo-blue.svg" alt="canvas blue logo" class="col-1 img-canvas ">
+      <nav class=" row col-5 light-grey-text justify-evenly align-center row1">
+        <a>Product</a>
+        <a>Implementations</a>
+        <a>About Us</a>
+        <a>Blog</a>
+        <div>
+          <button class=" white-text button-back-color contact-button font-A">Contact Us</button>
+      </nav>
+    </header>
+
+    <div class="hero-image  row-1 font-a white-text ">
+      <div class="overlay">
+        <p class="font-size-large font-abs1">The EMR Reimagined for Primary Care</p>
+        <p class="font-abs2"> Can't remember the last time you talked with a patient without focusing on the EMR? Canvas lets you rediscover joy in
+    the work you do. Capture meaningful data,identify gaps in care, and move seamlessly between tasks within the platform.
+    You stay present with patient while your whole care team stays up-to-speed. You stay present with patients while your
+    whole care team stays up-to-speed.Be a better doctor with Canvas, designed for primary care providers.
+        </p>
+        <br>
+        <button class=" white-text button-back-color get-started-button font-a font-abs3"> Get Started</button>
+      </div>
+    </div>
+
+    <div class=" clin light-grey-background row justify-content dark-grey-text font-a">
+      <div class="col-5 ">
+        <h2 class="text-align-center font-a-big"> Clinicians Prefer Canvas</h2>
+        <p class="text-align-center">Physician-owned primary care practices across
+    the country have already made Canvas the fabric of their care.</p>
+      </div>
+      <div class="clin-marg row justify-evenly align-center">
+        <div class="col-4">
+          <p>Cascade Family Practice,Portland OR </p>
+          <p>Temescal Creek Medicine,Oakland CA </p>
+          <p>San Juan Healthcare,Friday Harbor WA </p>
+        </div>
+        <div class=" col-4">
+          <p> Wards Medical Clinic , Bonifay FL</p>
+          <p> Caribou Medical, Towanda PA</p>
+          <p> Bay Tree Family Care, Red Bay AL</p>
+        </div>
+      </div>
```

Comment:
This `row` is a direct child of an element with the `row` class. You should never have a `row` immediately inside of another element with the `row` class. You should put this `row` into a `column` class element of some size.